pkg/receive/multitsdb.go

```diff
@@ -139,6 +136,60 @@ func NewMultiTSDB(
 	return mt
 }
 
+// getTenant returns the tenant with the given tenantID.
+// @testing: This method is only for testing purposes.
+func (t *MultiTSDB) getTenant(tenantID string) *tenant {
+	t.mtx.RLock()
+	defer t.mtx.RUnlock()
+	return t.tenants[tenantID]
+}
+
+// tsdbLocalClientsCopied returns a copy of tsdbClients.
+// @testing: This method is only for testing purposes.
+func (t *MultiTSDB) tsdbLocalClientsCopied() []store.Client {
```

Comment:
This is only called from one place. Let's inline it as per the coding style guide: https://thanos.io/tip/contributing/coding-style-guide.md/#keep-the-interface-narrow-avoid-shallow-functions